data/src/test/java/org/oppia/android/data/backends/gae/JsonPrefixNetworkInterceptorTest.kt

```diff
@@ -109,7 +109,7 @@ class JsonPrefixNetworkInterceptorTest {
       fun build(): TestApplicationComponent
     }
 
-    fun inject(networkInterceptorTest: JsonPrefixNetworkInterceptorTest)
-    fun inject(networkInterceptor: JsonPrefixNetworkInterceptor)
+    fun inject(jsonPrefixNetworkInterceptorTest: JsonPrefixNetworkInterceptorTest)
+    fun inject(jsonPrefixNetworkInterceptor: JsonPrefixNetworkInterceptor)
```

Comment:
This should be removed--only the test needs an injector generated.

Reply:
Done